lib/Zadm/Schema/KVM.pm

```diff
@@ -135,9 +135,8 @@ $SCHEMA = sub($self) {
     extra       => {
         optional    => 1,
         description => 'extra parameters',
-        default     => '',
         example     => '"extra" : "<parameters>"',
-        validator   => $self->sv->regexp(qr/^.+$/, 'expected a string'),
+        validator   => $self->sv->regexp(qr/^.*$/, 'expected a string'),
```

Comment:
empty string is fine ?

Reply:
Well generally empty values are ok for zone properties (e.g. `"bootargs" : "", ...`). I just double checked and it looks like it is not for `attr` properties:

```
[2022-02-03 18:57:25.31757] [10456] [debug] /usr/sbin/zonecfg -z kvmtest add attr ; set name = "extra" ; set value = "" ; set type = "string" ; end
value not specified
Resource incomplete.
```

Thanks for spotting it.